pkg/vmprovider/providers/vsphere2/constants/constants.go

```diff
@@ -32,8 +32,12 @@ const (
 	// EnableDiskUUIDExtraConfigKey Enable UUID ExtraConfig key.
 	EnableDiskUUIDExtraConfigKey = "disk.enableUUID"
 
-	// MMPowerOffVMExtraConfigKey ExtraConfig key to enable DRS to powerOff VMs when underlying host enters into
-	// maintenance mode. This is to ensure the maintenance mode workflow is consistent for VMs with vGPU/DDPIO devices.
+	// MMPowerOffVMExtraConfigKey ExtraConfig key to enable DRS to powerOff VMs
+	// when the underlying host enters into maintenance mode. This is to ensure
+	// the maintenance mode workflow is consistent for VMs with vGPU/DDPIO
+	// devices.
+	// Deprecated: Admins can power off the VMs as they would VMs that are not
+	//             managed by VM Operator.
```

Comment:
Hmm.  They _can_.  But the virtual machine controller in VM operator will immediately try to power on the VM, right?  I think if we go by this route, we need to have admins leverage the `pause-reconcile` annotation.

Reply:
Not necessary. Once a host has been marked for maintenance mode, a VM cannot be powered back onto that host. This is no different than DRS powering off the VM. We are just stating the admin will.

Reply:
Makes sense.  I was missing the fact that if a host is marked for MM, any attempt to power on the VM will place them on a different host automatically.  Thanks!

Reply:
I added the following to the PR description:

> Since it is not possible to power on a VM on a host marked for maintenance mode, attempts to power the VM on by VM Operator will force DRS to attempt to find another compatible host for the VM.